tests/test_modeling_common.py

```diff
@@ -1620,55 +1641,72 @@ def test_tied_weights_keys(self):
                     tied_params[i] = [p for p in tied_params[i] if re.search(key, p) is None]
 
             tied_params = [group for group in tied_params if len(group) > 1]
-            self.assertListEqual(tied_params, [])
+            self.assertListEqual(
+                tied_params,
+                [],
+                f"Missing `_tied_weights_keys` for {model_class}: add all of {tied_params} except one.",
+            )
 
-    def test_tied_model_weights_key_ignore(self):
-        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+    def test_model_weights_reload_no_missing_tied_weights(self):
```

Comment:
The previous test was mostly focused on checking tied weights were in the `_keys_to_ignore_on_load_missing` class variable, but we don't put them there anymore. It's thus adapted.